draft-ietf-sedate-datetime-extended.md

```diff
@@ -369,7 +369,7 @@ in email headers, described in {{Section 3.3 of RFC5322}} and introduced
 earlier in {{Section 3.3 of RFC2822}}.
 This email header convention is in actual use, while its adaptation into
 {{RFC3339}} always was
-handicapped by the fact that {{ISO8601-2000}} and later versions do not actually allow `-00:00`.
+held back by the fact that {{ISO8601-2000}} and later versions do not actually allow `-00:00`.
```

Comment:
The phrase "held back" suggests it never did get adapted into RFC3339, when in fact it does appear in RFC3339, and that's a problem.
Suggestion: ```its adaptation into {{RFC3339}} always was problematic because {{ISO8601-2000}} and later ...```

Reply:
> The phrase "held back" suggests it never did get adapted into RFC3339 

I don't have that other reading, so I'll have to ask you:

Which of these words would not have that connotation?

* hinder
* hamper
* set back
* hamstring
* encumber

Reply:
Well, it's more the combination with "adaptation" that's causing the other reading - most of the suggested alternatives still fail because "adaptation into ... was [word]" suggests it didn't get adapted into the standard.  I think only "encumber" survives that, but perhaps rephrasing away from "adaptation" might be the real fix.  Suggestion:

The email header convention is in actual use, while {{RFC3339}}'s adoption of it brought it into conflict with {{ISO8601-2000}} and later, which do not even allow '-00:00'.

Note, however, that I may have misunderstood a fragment that I'm seeing out of context in GitHub's review tool, so sanity-check with your better knowledge of context.

Reply:
(see below...)

Reply:
I guess "compromised" will do ;^>